apps/docs/.storybook/manager.ts

```diff
@@ -1,9 +1,9 @@
 import { addons } from '@storybook/manager-api';
-import theme from './theme';
-import './styles.css';
+import storybookTheme from './storybookTheme';
+import './manager.css';
 
 addons.setConfig({
-  theme: theme,
```

Comment:
Renamed this for clarity too, since we are introducing the idea of themes into our stories. This however is the theme of storybook itself.